### ISSUE TITLE
jquery autocomplete not displaying dynamic <ul> list when typing in text

### ISSUE THREAD

**Issue by kpreid** [google/caja#1589]
Original [issue 1589](https://code.google.com/p/google-caja/issues/detail?id=1589) created by steven.zeck on 2012-11-09T00:54:08.000Z:

What revision of the cajoler exhibits the problem?  On what browser and OS?  Chrome 23/Windows 7

<b>What steps will reproduce the problem?</b>
1. Copy/paste the below code into a GAS HTML file.
2. Create the HTML output from that file and go to your URL that runs it
3. Start typing in &quot;ja&quot;, it should display a list of possible results, but it doesn't.  If you keep &quot;ja&quot; typed in and use the down arrow key, the results &quot;scroll&quot; through.  If you copy/paste into a local HTML file (non-cajoled) it runs as expected.

<b>Please provide any additional information below.</b>

&lt;!DOCTYPE html&gt;
&lt;html&gt;
&lt;head&gt;
&lt;link href=&quot;http://ajax.googleapis.com/ajax/libs/jqueryui/1.8/themes/base/jquery-ui.css&quot; rel=&quot;stylesheet&quot; type=&quot;text/css&quot;/&gt;
&lt;script src=&quot;http://code.jquery.com/jquery-1.8.2.js&quot;&gt;&lt;/script&gt;
&lt;script src=&quot;http://code.jquery.com/ui/1.8.23/jquery-ui.js&quot;&gt;&lt;/script&gt;
&lt;script&gt;
$(function() {
    var availableTags = [
      &quot;ActionScript&quot;,
      &quot;AppleScript&quot;,
      &quot;Asp&quot;,
      &quot;BASIC&quot;,
      &quot;C&quot;,
      &quot;C++&quot;,
      &quot;Clojure&quot;,
      &quot;COBOL&quot;,
      &quot;ColdFusion&quot;,
      &quot;Erlang&quot;,
      &quot;Fortran&quot;,
      &quot;Groovy&quot;,
      &quot;Haskell&quot;,
      &quot;Java&quot;,
      &quot;JavaScript&quot;,
      &quot;Lisp&quot;,
      &quot;Perl&quot;,
      &quot;PHP&quot;,
      &quot;Python&quot;,
      &quot;Ruby&quot;,
      &quot;Scala&quot;,
      &quot;Scheme&quot;
    ];
    $( &quot;#tags&quot; ).autocomplete({
      source: availableTags
    });
  });
  &lt;/script&gt;
&lt;/head&gt;
&lt;body&gt;

&lt;div class=&quot;demo&quot;&gt;
&lt;div class=&quot;ui-widget&quot;&gt;
  &lt;label for=&quot;tags&quot;&gt;Tags: &lt;/label&gt;
  &lt;input id=&quot;tags&quot; /&gt;
&lt;/div&gt;
&lt;/div&gt;&lt;!-- End demo --&gt;

&lt;/body&gt;
&lt;/html&gt;
